CHANGES.md

```diff
@@ -7,6 +7,7 @@ number or not.
 ## Gameplay and UI
 
 - Default teams now have logos
+- International players from 85 countries, and better names for American players
```

Comment:
I think it's http://stackoverflow.com/a/26986320/786644 which seems crazy... can you either rebase or make a new PR, as that suggests? Sorry for the trouble. Bitbucket handles this automatically...